File: uikit/src/main/java/listeners/StickyHeaderAdapter.java
package listeners;

import android.view.ViewGroup;
import androidx.recyclerview.widget.RecyclerView;

public interface StickyHeaderAdapter<T extends RecyclerView.ViewHolder> {

    long getHeaderId(int var1);

    T onCreateHeaderViewHolder(ViewGroup var1);

    void onBindHeaderViewHolder(T var1, int var2, long var3);
}

File: app/src/main/java/com/inscripts/cometchatpulse/CustomView/StickyHeaderAdapter.kt
package com.inscripts.cometchatpulse.CustomView

import androidx.recyclerview.widget.RecyclerView
import android.view.ViewGroup

interface StickyHeaderAdapter<T: androidx.recyclerview.widget.RecyclerView.ViewHolder> {

     fun getHeaderId(var1: Int): Long

     fun onCreateHeaderViewHolder(var1: ViewGroup): T

     fun onBindHeaderViewHolder(var1: T, var2: Int, var3: Long)
}